Recipes/InheritedTypeJsonConverter/InheritedTypeJsonConverter.cs

```diff
@@ -239,5 +194,69 @@ private IEnumerable<Type> GetChildTypes(Type type)
 
             return allChildTypes[type];
         }
+
+        private IEnumerable<CandidateChildType> GetCandidateChildTypes(Type objectType, HashSet<string> jsonProperties)
+        {
+            var childTypes = GetChildTypes(objectType);
```

Comment:
agreed - and then GetCandidateChildTypes can become static.  Per our convo, this is NOT chainable because I am unable to make this an extension method.